src/types/promotedPublications.types.ts

```diff
@@ -0,0 +1,4 @@
+export interface PromotedPublications {
```

Comment:
Teknisk sett representerer dette objektet en persons preferanser, og promotedPublications er foreløpig eneste _property_ der, eller?
```suggestion
export interface PersonPreferences {
```